sources/packages/forms/src/form-definitions/sfaa2025-26.json

```diff
@@ -8598,7 +8582,22 @@
                 "json": ""
               },
               "customConditional": "",
-              "logic": [],
+              "logic": [
+                {
+                  "name": "Check Youth In Care Options",
```

Comment:
The follow up question doesn't seem to populate for PT for 2025-2026 PY form. Let me know if I am missing something.
![image](https://github.com/user-attachments/assets/63e9fdde-583f-4950-88ee-a171ff11b156)